tools/sense_studio/templates/testing.html

```diff
@@ -0,0 +1,136 @@
+    <div class="uk-text-meta uk-text-large uk-margin-medium-bottom">
+        <span uk-icon="icon: folder"></span> {{ path }}
+    </div>
+    <form class="uk-form-stacked" method="post">
+        <input type="hidden" id="project" name="project" value="{{ project }}">
+        <input type="hidden" id="path" name="path" value="{{ path }}">
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="classifier">Custom Classifier</label>
+                 <div class="uk-form-controls">
+                    <select class="uk-select {{ 'uk-form-danger' if not classifiers }}" id="classifier">
+                        {% if classifiers %}
+                            {% for classifier in classifiers %}
+                                <option>{{ classifier }}</option>
+                            {% endfor %}
+                        {% else %}
+                            <option value="">No classifiers to select from</option>
+                        {% endif %}
+                    </select>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-2@s uk-grid-small" uk-grid>
+            <div>
+                <label class="uk-form-label" for="title">Window Title</label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: pencil"></span>
+                        <input class="uk-input" type="text" id="title" placeholder="Title" autocomplete="off" />
+                    </div>
+                </div>
+            </div>
+            <div class="uk-margin-top">
+                <label class="uk-form-label">Choose Input Source</label>
+                 <div class="uk-form-controls">
+                     <div class="uk-inline uk-width-1-1">
+                        <label>
+                            <input class="uk-radio" type="radio" name="inputSource" value="0" onclick="toggleInputVideoField();" checked>
+                            &nbsp;&nbsp;Use Webcam
+                        </label>
+                        <label class="uk-margin-large-left">
+                            <input class="uk-radio" type="radio" name="inputSource" value="1" onclick="toggleInputVideoField();">
+                            &nbsp;&nbsp;Input Video
+                        </label>
+                     </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                 <div class="uk-form-controls">
+                     <div class="uk-width-1-1">
+                        <label>
+                            <input class="uk-checkbox" type="checkbox" name="saveVideo" onclick="toggleOutputVideoField();">
+                            &nbsp;&nbsp;Save Output Video
```

Comment:
Spontaneous idea on the output video name: How about we just name it automatically, similar to how we do it in the video recording scripts. Then we don't need to fill an extra field.